CHANGELOG.md

```diff
@@ -10,8 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bugfixes
 
+* Add missing mapping for azure/metrics input. [#1474](https://github.com/elastic/package-registry/pull/1474)
```

Comment:
Added as a bugfix this PR https://github.com/elastic/package-registry/pull/1474
Is that ok?

Reply:
i just realize i forgot to add a changelog on my pr https://github.com/elastic/package-registry/pull/1482
could we add it here or i open a new pr? 🤓 

`* Allow empty package paths on startup  [1482](https://github.com/elastic/package-registry/pull/1482)`

Reply:
Sure, I'll add it as part of this PR :+1:

Reply:
Sure, I'll add it as part of this PR :+1: